python/openmldb/dbapi/dbapi.py

```diff
@@ -18,8 +18,8 @@
 sys.path.append("../")
 
 import logging
-from openmldb_sdk import sdk as sdk_module
-from openmldb_native import sql_router_sdk
+from ..sdk import sdk as sdk_module
+from ..native import sql_router_sdk
```

Comment:
Make sure this works in linux and macos environments.

Reply:
This is relative import and should work in Linux and Macos:
### If the import module in the same dir, use e.g: from . import core
### If the import module in the top dir, use e.g: from .. import core
### If the import module in the other subdir, use e.g: from ..other import core